### ISSUE TITLE
Bug while opening SQL Premium questions

### ISSUE THREAD

**Issue by VinothKanna007** [31b4/Leetcode-Premium-Bypass#22]
### Bug in SQL Premium Questions

1. Select Database
2. click settings
3. select premium qns
4. select any one question (it is not opening)

![image](https://github.com/31b4/Leetcode-Premium-Bypass/assets/117466360/a2e93424-3a2e-4337-8919-5d21472ef892)

**Comment by VinothKanna007** [comment]
This is how it opens. PFA

![image](https://github.com/31b4/Leetcode-Premium-Bypass/assets/117466360/ddb6f855-a8c5-4cb1-8349-49140f57f97c)

**Comment by 31b4** [comment]
Unfortunately, this is how i have that in the database, im gonna think about this issue but i dont think i can solve it.
<img width="1399" alt="image" src="https://github.com/31b4/Leetcode-Premium-Bypass/assets/75566095/374a243e-db28-42c9-aba7-676f6b5d576f">

**Comment by VinothKanna007** [comment]
I'm looking only for SQL premium questions. Do we have any alternate ?

**Comment by 31b4** [comment]
> I'm looking only for SQL premium questions. Do we have any alternate ?

If you can name a specific question, i can ss for you, @31b4 my dc.

**Comment by VinothKanna007** [comment]
Basically, I want to try and test the ouput in the leetcode itself for all the sql **premium** problems.

**Comment by 31b4** [comment]
> Basically, I want to try and test the ouput in the leetcode itself for all the sql **premium** problems.

yes, for that you have to buy premium, its not possible with any extension
